### ISSUE TITLE
Dark text on dark background with both dark themes

### ISSUE THREAD

**Issue by pickelmikel** [pgadmin-org/pgadmin4#6355]
Greetings.  I noticed this when updating from 6.20 to 6.21.

Using either dark theme produces this anomaly as it was displaying correctly in 6.20.
Dark themes had white text on dark background.  I have noticed it with all versions since 6.20.

Attached is a screenshot of a table from the current version, 7.2 that was displaying the text correctly in 6.20. 

The open dialog shows there is text in the cell; however, it is dark text on dark background.
Some of the text is showing correctly; was there a change to the way pgAdmin displays certain types,  like arrays? 

![dark_theme_text_is_dark](https://github.com/pgadmin-org/pgadmin4/assets/88795708/90934c60-f369-4446-b9eb-4c52e5f6f4d9)

Thanks to everyone who has made this software possible!

**Comment by adityatoshniwal** [comment]
@pickelmikel Where is issue exactly? We're not able to figure it out. Colors seems to be fine. The screenshot shows white text on dark background.

**Comment by pickelmikel** [comment]
@adityatoshniwal Thank you for the reply.  I am trying to find a copy of pgAdmin 6.20 to try and see why my query output had changed after updating.  Apparently from what I see is that there is a carriage return after the date, which is why in the cell right of the dates looks blank.  Am trying to remember if I changed something to make that happen; although I do remember updating from 6.20 to 6.21 and noticed it, then reverted back and was displaying correctly. I would like to get a copy of 6.20 to try the query again.  I am looking for a copy of 6.20 in .deb; however, I am unable to locate it.  If someone has a link to 6.20 I would like to check it out again. 

Long story short, the text is displaying correctly.  Somehow carriage returns were put in and that's why the cells "look" blank.

This bug report is closed as the display IS showing correctly

Thank you for getting back to me, I appreciate everyone for their help!
